### DISCUSSION TITLE
Cannot expand step after browser refresh

### DISCUSSION THREAD

**Discussion by ro0NL** [community/community#55605] [Actions]
### Select Topic Area

Product Feedback

### Body

When a step is running it's initially expanded, thus i can follow realtime output.

On browser refresh the step is collapsed, and i cannot expand it anymore, only after the step finishes, thus i cannot follow realtime output anymore

I'm using Google Chrome Version 113.0.5672.92

**Comment by JesusCastroFernandez** [comment]
Lamentablemente, la funcionalidad de expansión y contracción de los pasos en la interfaz del navegador no está controlada por el modelo de lenguaje en sí, sino por la implementación y configuración específica del entorno de uso. En este caso, se trata de la interfaz de usuario del navegador que estás utilizando (Google Chrome).

La forma en que los pasos se expanden y contraen puede variar según la configuración y la versión del navegador. Es posible que las actualizaciones del navegador o cambios en la configuración puedan afectar la funcionalidad de expansión y contracción de los pasos en la interfaz.

Te sugiero verificar si hay alguna opción de configuración o ajuste en la configuración del navegador que pueda afectar la visualización de los pasos en tiempo real. También puedes considerar probar con otros navegadores o versiones diferentes para ver si obtienes resultados diferentes en términos de la funcionalidad de expansión y contracción de los pasos.

Si el problema persiste o necesitas asistencia adicional, te recomiendo ponerse en contacto con el soporte técnico del navegador o buscar en foros y comunidades en línea donde los usuarios puedan compartir soluciones o experiencias similares.
